### PR TITLE
Update README (updated with @waldyrious pr too)

### PR DESCRIPTION
**Pull request by kavin25**
- Add fzf link
- Add username usage instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
-gh extension install hashue/gh-fuzzyclone
+You can also pass in a username like so
+
+```sh
+gh fuzzyclone hashue
 ```
 
+## Installation
+
+`gh extension install hashue/gh-fuzzyclone`
```
Comment (hashue) [inline README.md:24]: I'd like you to unify to "code block"

Comment (hashue) [inline README.md:24]: I'd like you to unify to "code block"

**Review by hashue — CHANGES_REQUESTED** [review]
Thanks for PR. I  added some "Request changes".

File: README.md (hunk 1 above)
Comment (kavin25) [reply README.md]: Done @hashue

**Comment by hashue** [comment]
Thanks! merged.

**Pull request merged**